.github/workflows/lint-test.yaml

```diff
@@ -9,10 +9,20 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
 
+      # Only build a kind cluster if there are chart changes to test.
+      # To-do: make this more robust. Perhaps with 'ct list-changed', so we can
+      # get remote branch and charts dir from ct config.
+      - name: Check changed charts
+        id: check_charts
+        run: |
+          changes=$(git diff --find-renames --name-only "$(git rev-parse --abbrev-ref HEAD)" remotes/origin/master -- charts)
+          [ -n "$changes" ] && echo "::set-output name=changed::true" || echo "No chart changes"
```

Comment:
I think that special kind of log format only works if you log with their toolkit from a JS action?

Reply:
Seems it does work, but you'd have to skip all subsequent steps.

Reply:
Yeah their documentation wasn't 100% clear on that to me either, but I found a fun issue about it with tests I was able to emulate: https://github.com/actions/toolkit/issues/96. I have tested it, and it only skips the correct step, not all subsequent ones. I left the multiple "revert" chart change commits in this PR just to show how it didn't work at points then worked with this specific syntax 😄

Reply:
Oh, did you mean we'd also have to add a similar check to the subsequent steps as well?
> Seems it does work, but you'd have to skip all subsequent steps.

If so, I didn't add that because ct `lint` and `install` already do this check.